### PR TITLE
MGMT-1433 User-Auth pass Pull Secret Token as env param

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

File: internal/host/installcmd.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func (i *installCmd) GetStep(ctx context.Context, host *models.Host) (*models.St
 	cmdArgsTmpl := "sudo podman run -v /dev:/dev:rw -v /opt:/opt:rw --privileged --pid=host --net=host " +
 		"-v /var/log:/var/log:rw --name assisted-installer {{.INSTALLER}} --role {{.ROLE}} --cluster-id {{.CLUSTER_ID}} --host {{.HOST}} " +
 		"--port {{.PORT}} --boot-device {{.BOOT_DEVICE}} --host-id {{.HOST_ID}} --openshift-version {{.OPENSHIFT_VERSION}} " +
-		"--controller-image {{.CONTROLLER_IMAGE}}"
+		"--controller-image {{.CONTROLLER_IMAGE}} --env PULL_SECRET_TOKEN"
```
Comment (eranco74) [inline internal/host/installcmd.go:55]: ```--env PULL_SECRET_TOKEN```must come before the image name {{.INSTALLER}}

**Comment by rollandf** [comment]
@ronniel1 @ori-amizur @eranco74 Can you please review?

**Comment by rollandf** [comment]
> LGTM
> Guess you are adding the env to the install command in another commit

What do you mean?

**Comment by eranco74** [comment]
> > LGTM
> > Guess you are adding the env to the install command in another commit
> 
> What do you mean?

OK I missed it.
https://github.com/filanov/bm-inventory/pull/513/files#diff-4bc7e9b2843bafc6870217476b7be864R53

**Pull request merged**